source/test/F0.Minesweeper.Logic.Tests/Minelayer/FirstEmptyMinelayerTest.cs

```diff
@@ -0,0 +1,75 @@
+using System.Collections.Generic;
+using F0.Minesweeper.Logic.Abstractions;
+using F0.Minesweeper.Logic.Minelayer;
+using FluentAssertions;
+using Xunit;
+
+namespace F0.Minesweeper.Logic.Tests.Minelayer
+{
+	public class FirstEmptyMinelayerTest
+	{
+		[Theory]
+		[InlineData(0, 0)]
+		[InlineData(0, 1)]
+		[InlineData(1, 1)]
+		[InlineData(1, 3)]
+		[InlineData(3, 1)]
+		[InlineData(3, 4)]
+		[InlineData(4, 3)]
+		public void PlaceMines_WithBadLuck_ResultsInClickedLocationIsNoMine(uint clickedLocationX, uint clickedLocationY)
+		{
+			Location clickedLocation = new(clickedLocationX, clickedLocationY);
+			LocationShufflerToTest locationShuffler = new(mineLocations);
+			FirstEmptyMinelayer minelayerUnderTest = new(locationShuffler);
```

Comment:
we could avoid the cast to `IMinelayer` in the next line with:
`IMinelayer minelayerUnderTest = new FirstEmptyMinelayer(locationShuffler);`
here in this file and in the other minelayer-test-fixtures

Reply:
Sounds good! I changed this.